doc/specs/stdlib_linalg.md

```diff
@@ -989,4 +989,121 @@ Exceptions trigger an `error stop`, unless argument `err` is present.
 ```fortran
 {!example/linalg/example_svdvals.f90!}
 ```
+## `.inv.` - Inverse operator of a square matrix
 
+### Status
+
+Experimental
+
+### Description
+
+This operator returns the inverse of a `real` or `complex` square matrix \( A \).
+The inverse \( A^{-1} \) is defined such that \( A \cdot A^{-1} = A^{-1} \cdot A = I_n \).
+
+This interface is equivalent to the function  [[stdlib_linalg(module):inv(interface)]]. 
+
+### Syntax
+
+`b = ` [[stdlib_linalg(module):operator(.inv.)(interface)]] `a`
+
+### Arguments
+
+`a`: Shall be a rank-2 square array of any `real` or `complex` kinds. It is an `intent(in)` argument.
+
+### Return value
+
+Returns a rank-2 square array with the same type, kind and rank as `a`, that contains the inverse of `a`.
+
+If an exception occurred on input errors, or singular matrix, NaNs will be returned.
```

Comment:
Maybe just add something like: "For fine-grained `error` control in case of singular matrices prefer the `subroutine` interface."
Just to be extra explicit

Reply:
Thank you @jalvesz, it's a good idea! [a292e9d](https://github.com/fortran-lang/stdlib/pull/828/commits/a292e9df496d5b048e249b898c25b7d3513d1cd4)